src/components/header/header.tsx

```diff
@@ -33,9 +35,22 @@ export const Header = ({
 
   const [navToggled, setNavToggled] = useRecoilState(navToggledState);
 
-  const userClient = useUserClient();
+  const userClient = useAccountClient();
```

Comment:
should we rename the variable to `accountClient` since we're adding a user client later?

Reply:
haha, good point. I'll get those fixed